applications/nrf5340_audio/src/audio/streamctrl_broadcast_sink.c

```diff
@@ -256,6 +256,21 @@ static void le_audio_msg_sub_thread(void)
 				ERR_CHK(ret);
 			}
 
+			if (IS_ENABLED(CONFIG_BT_OBSERVER)) {
```

Comment:
This looks good to me @rick1082 :) Could you please give a walkthrough in our next standup?